### PR TITLE
Support for task parameters:

### PR DESCRIPTION
**Pull request by toudi**
- Support for variable-length arguments
- Support for keyword arguments
- Backward compatibility: one argument only

### CODE REVIEW

**Comment by toudi** [comment]
Sorry it took a while, but i hope it was worth it ;)

File: README.md [hunk 1]
```diff
@@ -46,6 +46,83 @@ the task name, by specifying `name` parameter of the decorator. Here's how:
...
+    client.submit_job('my-task-name', kwargs={"background": True, "unique": False})
+    client.submit_job('my-task-name', args=[True,False])
+    
+Now, for the final stride:
+
+    client.submit_job('my-task-name', background=True, unique=True, kwargs={"background": False, "unique": False})
+    
+Don't panic, your task is safe! That's because you're using ``kwargs`` directly. Therefore,
+gearman's bindings would recieve ``True`` for ``submit_job`` function, whether your task would
+recieve ``False``.
```
Comment (fwenzel) [inline README.md:116]: This is a great design, I think. Good job.

File: django_gearman/models.py [hunk 2]
```diff
@@ -27,14 +27,56 @@ def __init__(self, **kwargs):
...
+    
+    def parse_data(self, arg, args = [], kwargs = {}, *arguments, **karguments):
+        data = {
+            "args": [],
+            "kwargs": {}
+        }
+        
+        """
+        The order is significant:
+        - First, use pythonic *args and/or *kwargs
```
Comment (fwenzel) [inline django_gearman/models.py:39]: **kwargs (double *)

File: django_gearman/models.py [hunk 3]
```diff
@@ -27,14 +27,56 @@ def __init__(self, **kwargs):
...
+    Re-implement the submit_job function, in order to handle *args and **kwargs
+    """
+    def submit_job(self, task, orig_data = None, unique=None, priority=None,
+                   background=False, wait_until_complete=True, max_retries=0,
+                   poll_timeout=None, args=[], kwargs={}, *arguments, **karguments):
+        
+        data = self.parse_data(orig_data, args, kwargs, *arguments, **karguments)
+        
+        return super(DjangoGearmanClient, self).submit_job(
+            task, data, unique, priority, background, wait_until_complete, max_retries, poll_timeout
```
Comment (fwenzel) [inline django_gearman/models.py:64]: You should line-break at about 79 characters: http://www.python.org/dev/peps/pep-0008/

File: django_gearman/models.py [hunk 4]
```diff
@@ -27,14 +27,56 @@ def __init__(self, **kwargs):
...
+        )
```
Comment (fwenzel) [inline django_gearman/models.py:65]: You don't have to have the closing parenthesis on a line by itself. Fine to put it at the end of the previous one.

**Comment by fwenzel** [comment]
Good job! I'll land this later today and I'll iron out some nits, but generally this is great.

**Pull request merged**